lib/t_web/views/view_helpers.ex

```diff
@@ -140,7 +142,33 @@ defmodule TWeb.ViewHelpers do
 
   defp process_label(label), do: label
 
-  defp s3_key_urls(key, width) when is_binary(key) do
+  defp s3_key_image_url(key, width) when is_binary(key) do
```

Comment:
it returns a map, not a single url, that was the reason behind `urls` in the name